content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -761,6 +761,45 @@ Default values must be pruned (with the exception of defaults for `metadata` fie
 
 Default values for `metadata` fields of `x-kubernetes-embedded-resources: true` nodes (or parts of a default value covering `metadata`) are not pruned during CustomResourceDefinition creation, but through the pruning step during handling of requests.
 
+#### Defaulting and Nullable
+
+**New in 1.20:** null values for fields that either don't specify the nullable flag, or give it a false value, will be pruned before defaulting happens. If a default is present, it will be applied. When nullable is true, null values will be conserved and won't be defaulted.
```

Comment:
I know I wrote this sentence, but in retrospect "false value" sounds weird. What do you think of:
> `false` value.

Or happy to see another suggestion!

Reply:
+1. Even something like "set `nullable: false`" could also work since we could also replace "when nullable is true" with "when `nullable: true`" is set.